senders/client_factory.go

```diff
@@ -44,6 +45,18 @@ type configuration struct {
 	SDKMetricsTags       map[string]string
 }
 
+func (c *configuration) Direct() bool {
```

Comment:
I am not sure about the naming convention that we follow, but usually, in java, we use something like `isDirectIngested`.

Reply:
Thanks for catching this. Although it is allowed to write methods like GetXXX, it usually isn't done that way in GO.  See https://go.dev/doc/effective_go#Getters.  I am assuming that not having GetXXX methods in Go carries over to not having IsXXX methods in Go as well.